scripts/build-kmm.js

```diff
@@ -0,0 +1,61 @@
+const path = require('path');
+const fs = require('fs');
+const { deduplicate } = require('./utils');
+
+const rootPath = 'android/src/commonMain/kotlin/com/stremio/translations';
+const stringsPath = path.join(rootPath, 'strings');
+const packageName = 'com.stremio.translations';
+
+fs.mkdirSync(path.join(process.cwd(), rootPath), { recursive: true });
+fs.mkdirSync(path.join(process.cwd(), stringsPath), { recursive: true });
+
+const files = fs.readdirSync(process.cwd()).filter(file => /[a-z]{2}-[A-Z]{2}\.json$/g.test(file));
+
+const escape = str => str
+  .replaceAll('\\', '\\\\')
+  .replaceAll('"', '\\"')
+  .replaceAll('\n', '\\n')
+  .replaceAll('%', '%%')
+  .replaceAll(/\${\d+}/gm, '%s');
+
+function writeDataClass() {
+  const defaultTranslation = JSON.parse(fs.readFileSync(path.join(process.cwd(), 'en-US.json')));
+  let stringClass = `package ${packageName}\n\n`;
+  stringClass += 'internal data class Strings(\n';
+  const dedup = deduplicate(defaultTranslation);
+  Object.keys(dedup).forEach(key => stringClass += `  val ${key}: String,\n`);
+  stringClass += ')';
+  fs.writeFileSync(path.join(process.cwd(), rootPath, 'Strings.kt'), stringClass);
+}
+
+function writeLocales() {
+  let stringClass = `package ${packageName}\n\n`;
+  stringClass += 'object Locales {\n';
+  const locales = new Set(files.map(file => file.split('-')[0]));
```

Comment:
Why are you taking only the country part of it? You should use the full language code, as you'll loose some languages (ie pt-PT and pt-BR)?

Reply:
I was like "I need to split this to get the locale only", but that you're right that doesn't make sense. It should be the full language code indeed.